rnnoise/rnnoise.js

```diff
@@ -0,0 +1,139 @@
+'use strict';
+
+import {buildConstantByNpy} from '../common/utils.js';
+
+export class RNNoise {
+  constructor(modelPath, batchSize, frames) {
+    this.baseUrl_ = modelPath;
+    this.batchSize_ = batchSize;
+    this.frames_ = frames;
+    this.model_ = null;
+    this.graph_ = null;
+    this.builder_ = null;
+  }
+
+  async load() {
+    const context = navigator.ml.createContext();
+    this.builder_ = new MLGraphBuilder(context);
+    // Create constants by loading pre-trained data from .npy files.
+    const inputDenseKernel0 = await buildConstantByNpy(this.builder_,
+        this.baseUrl_ + 'input_dense_kernel_0.npy');
+    const inputDenseBias0 = await buildConstantByNpy(this.builder_,
+        this.baseUrl_ + 'input_dense_bias_0.npy');
+    const vadGruW = await buildConstantByNpy(this.builder_,
+        this.baseUrl_ + 'vad_gru_W.npy');
+    const vadGruR = await buildConstantByNpy(this.builder_,
+        this.baseUrl_ + 'vad_gru_R.npy');
+    const vadGruBData = await buildConstantByNpy(this.builder_,
+        this.baseUrl_ + 'vad_gru_B.npy');
+    const noiseGruW = await buildConstantByNpy(this.builder_,
+        this.baseUrl_ + 'noise_gru_W.npy');
+    const noiseGruR = await buildConstantByNpy(this.builder_,
+        this.baseUrl_ + 'noise_gru_R.npy');
+    const noiseGruBData = await buildConstantByNpy(this.builder_,
+        this.baseUrl_ + 'noise_gru_B.npy');
+    const denoiseGruW = await buildConstantByNpy(this.builder_,
+        this.baseUrl_ + 'denoise_gru_W.npy');
+    const denoiseGruR = await buildConstantByNpy(this.builder_,
+        this.baseUrl_ + 'denoise_gru_R.npy');
+    const denoiseGruBData = await buildConstantByNpy(this.builder_,
+        this.baseUrl_ + 'denoise_gru_B.npy');
+    const denoiseOutputKernel0 = await buildConstantByNpy(this.builder_,
+        this.baseUrl_ + 'denoise_output_kernel_0.npy');
+    const denoiseOutputBias0 = await buildConstantByNpy(this.builder_,
+        this.baseUrl_ + 'denoise_output_bias_0.npy');
+    // Build up the network.
+    const input = this.builder_.input(
+        'input', {type: 'float32', dimensions: [this.batchSize_,
+          this.frames_, 42]});
+    const inputDense0 = this.builder_.matmul(input, inputDenseKernel0);
+    const biasedTensorName2 = this.builder_.add(inputDense0, inputDenseBias0);
+    const inputDenseTanh0 = this.builder_.tanh(biasedTensorName2);
+    const vadGruX = this.builder_.transpose(
+        inputDenseTanh0, {permutation: [1, 0, 2]});
+    // **********hiddenSize = 24**********
+    const vadGruB = this.builder_.slice(
+        vadGruBData, [0], [3 * 24], {axes: [1]});
```

Comment:
could you please create constant like `const hiddenSize = 24`? And use it instead of use number directly.

Reply:
Modified according to your suggestion.